apps/web/src/app/dash/layout.tsx

```diff
@@ -10,18 +10,35 @@ import ProfileButton from "@/components/shared/ProfileButton";
 import ClientToast from "@/components/shared/ClientToast";
 
 import { TRPCReactProvider } from "@/trpc/react";
+import { db } from "db";
+import { eq } from "db/drizzle";
+import { users } from "db/schema";
 
 interface DashLayoutProps {
 	children: React.ReactNode;
 }
 
 export default async function DashLayout({ children }: DashLayoutProps) {
-	const user = await currentUser();
+	const clerkUser = await currentUser();
 
-	if (!user || !user.publicMetadata.registrationComplete) {
+	if (!clerkUser || !clerkUser.publicMetadata.registrationComplete) {
```

Comment:
I thought we were saying we were going to remove this metadata tag all together and just check their presence in the db